kolibri/plugins/coach_tools/yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
What does this lock file do? Doesn't seem like there's anything important specified here.